tests/automation/disappearing_messages.spec.ts

```diff
@@ -222,3 +225,153 @@ test_Alice_2W(
+        'disappear-control-message',
+        englishStrippedStr('disappearingMessagesSet')
+          .withArgs({
+            name: alice.userName,
+            time: '10 seconds',
+            disappearing_messages_type: englishStrippedStr(
+              'disappearingMessagesTypeSent',
+            ).toString(),
+          })
+          .toString(),
+      ),
+    ]);
+    await sendMessage(aliceWindow1, testMessage);
+    // Check message has appeared in receivers window and linked device
+    await Promise.all([
+      waitForTextMessage(bobWindow1, testMessage),
+      waitForTextMessage(aliceWindow2, testMessage),
+    ]);
+    await clickOnTestIdWithText(
+      aliceWindow1,
+      'conversation-options-avatar',
+      undefined,
+      undefined,
+      1000,
+    );
+    await clickOnElement({
+      window: aliceWindow1,
+      strategy: 'data-testid',
+      selector: 'disappearing-messages',
+      maxWait: 100,
+    });
+    await clickOnElement({
+      window: aliceWindow1,
+      strategy: 'data-testid',
+      selector: 'disappear-off-option',
+    });
+    await clickOnElement({
+      window: aliceWindow1,
+      strategy: 'data-testid',
+      selector: 'disappear-set-button',
+    });
+    // Select Follow setting in Bob's window
+    await clickOnMatchingText(
+      bobWindow1,
+      englishStrippedStr('disappearingMessagesFollowSetting').toString(),
+    );
+    await clickOnElement({
+      window: bobWindow1,
+      strategy: 'data-testid',
+      selector: 'session-confirm-ok-button',
+    });
+    // Check control message are visible and correct
+    // Each window has two control messages: You turned off and other user turned off (because we're following settings)
+    await Promise.all([
+      waitForTestIdWithText(
+        aliceWindow1,
+        'disappear-control-message',
+        englishStrippedStr('disappearingMessagesTurnedOffYou').toString(),
+      ),
+      waitForTestIdWithText(
+        aliceWindow1,
+        'disappear-control-message',
+        englishStrippedStr('disappearingMessagesTurnedOff')
+          .withArgs({ name: bob.userName })
+          .toString(),
+      ),
+      waitForTestIdWithText(
+        aliceWindow2,
+        'disappear-control-message',
+        englishStrippedStr('disappearingMessagesTurnedOffYou').toString(),
+      ),
+      waitForTestIdWithText(
+        aliceWindow2,
+        'disappear-control-message',
+        englishStrippedStr('disappearingMessagesTurnedOff')
+          .withArgs({ name: bob.userName })
+          .toString(),
+      ),
+      waitForTestIdWithText(
+        bobWindow1,
+        'disappear-control-message',
+        englishStrippedStr('disappearingMessagesTurnedOff')
+          .withArgs({ name: alice.userName })
+          .toString(),
+      ),
+      waitForTestIdWithText(
+        bobWindow1,
+        'disappear-control-message',
+        englishStrippedStr('disappearingMessagesTurnedOffYou').toString(),
+      ),
+    ]);
```

Comment:
I'm wanting to create a function that separates out the control message checks but thought I would do it in a different PR. There are alot of control messages and its kinda clogging up the test.